src/approaches/grammar_search_invention_approach.py

```diff
@@ -653,7 +653,8 @@ def _evaluate_with_operators(self, predicates: FrozenSet[Predicate],
             ground_ops = {op for strips_op in strips_ops for op
```

Comment:
Btw, why does grammar search get noticeably faster on painting with this PR? Is it bc of the reachable_nsrts change?

Reply:
I don't think it's that, because I'm not using reachable_nsrts in the grammar search (because I was nervous, I think in theory it should be fine...). It might be because of all the filtering out of static atoms that I'm doing now, which we weren't doing before?

Reply:
hmm, maybe we should use it there too... right?